build

```
@@ -8,6 +8,7 @@ try {
 	process.exit(1);
 }
 
+var {transform} = require('sucrase');
```

```suggestion
var transform = require('sucrase').transform;
```

(Destructuring is also not supported by early Node versions)

---

Ok. What is the minimum version we are supporting? Let me know so I can test.